packages/@webex/plugin-meetings/src/meeting/request.ts

```diff
@@ -792,4 +792,25 @@ export default class MeetingRequest extends StatelessWebexPlugin {
       }
     });
   }
+
+  /**
+   * Make a network request to enable or disable reactions.
+   * @param {boolean} options.enable - determines if we need to enable or disable.
+   * @param {locusUrl} options.locusUrl
+   * @returns {Promise}
+   */
+  toggleReactions({enable, locusUrl, requestingParticipantId}: {enable: boolean, locusUrl: string, requestingParticipantId: string}) {
```

Comment:
minor: please create a type for the parameter

for example:
```suggestion
type ToggleReactionsOptions = {
  enable: boolean, 
  locusUrl: string, 
  requestingParticipantId: string
}
  toggleReactions({enable, locusUrl, requestingParticipantId}: ToggleReactionsOptions) {
```

Reply:
I would avoid creating new conventions, I just followed the existing examples.
I do agree with you that it's better to extract it away, but until we agree on new typescript conventions on the SDK, I'll leave it as is.

Reply:
I would recommend adding the types as @maxinteger mentioned 

i am working on generating the d.ts file for all the code once that done we can move the types to those files

Reply:
Well, I can't really do that. It's complaining i can't create a type inside a class.
I added a new file called request.type.ts in which i added it.